pkg/test/fixtures/fixtures.go

```diff
@@ -44,6 +45,10 @@ const (
 	AccessPolicyApp           = "allowedAccessApp"
 )
 
+const ApplicationName = "myapplication"
+const ApplicationNamespace = "mynamespace"
+const ApplicationTeam = "myteam"
```

Comment:
Hvorfor er disse tre utenfor const-blokken ovenfor? Kunne ikke merget de, og endret kommentaren?

Reply:
The const block above has been removed and these three variables have replaced that block.